templates/pattern-localfooter--layout.html.twig

```diff
@@ -0,0 +1,247 @@
+ *                        by a common macro or arbitrary html markup.
+ * - secondary_links:     Either a structured array of links that get rendered
+ *                        by a common macro or arbitrary html markup.
+ * - signup_form_content: An open variable for placing content above the form.
+ * - signup_form_action:  The url that the form should submit to.
+ * - signup_form_method:  The http method in which the form should use (get|post)
+ * - signup_form_field_email: The name and id of the email field.
+ * - signup_form_field_email_placeholder: The placeholder text for the field.
+ * - signup_form_field_submit: The name and id of the submit button.
+ * - signup_form_field_submit_value: The value of the submit button.
+ * - signup_form_fields:  Option variable that allows you to put any additional
+ *                        markup in to the signup form. This could be content or
+ *                        additional fields.
+ * - weblogin_url:        Optional - The url in which a user goes to login with.
+ * - weblogin_text:       The text to display for the weblogin button link.
+ */
+#}
+{%- if template_path_lockup is empty -%}
+  {%- set template_path_lockup = "@decanter/components/lockup/lockup.twig" -%}
+{%- endif %}
+
+{%- if lockup_options is empty -%}
+  {%- set lockup_options = { "modifier_class": 'su-lockup--option-a', "line1": lockup_title, "attributes": attributes|without('id') } -%}
+{%- endif -%}
+
+{%- if template_path_link is empty -%}
+  {%- set template_path_link = "@decanter/components/link/link.twig" -%}
+{%- endif %}
+
+{%- if template_path_signup_form is empty -%}
+  {%- set template_path_signup_form = "@decanter/components/signup-form/signup-form.twig" -%}
+{%- endif %}
+
+{%- if template_path_macro is empty -%}
+  {%- set template_path_macro = "@decanter/utilities/macros/link-list.twig" -%}
+{%- endif -%}
+{%- import template_path_macro as list_macro -%}
+
+<div {{ attributes }} class="su-local-footer {{ modifier_class }}">
+  {# The header is completely optional and won't render if there isn't any content #}
+  {% if lockup_title is not empty or weblogin_url is not empty or headercontent is not empty %}
+    <div class="su-local-footer__header">
+      {# Web login button #}
+      {% if weblogin_url is not empty %}
+        {% include template_path_link with {
+          "modifier_class": "su-link--internal",
+          "link_href": weblogin_url,
+          "link_content": weblogin_text|default('Web Login')
+        } only %}
+      {% endif %}
+      {# Lockup template. Set usually to option-a #}
+      {% include template_path_lockup with lockup_options %}
+      {# Adding the school of Medicine logo for BioEngineering #}
+      {% if show_su_med_logo is same as(1) %}
+        <div class="su-med-logo">
+          <a href="https://med.stanford.edu/">
+            <img src="/themes/custom/soe_basic/dist/assets/img/stanford_medicine.png" alt="Stanford Medicine logo"/>
```

Comment:
This path won't always be correct. You might need to pass the path from the preprocess using `drupal_get_path`

Reply:
I changed it. Is this what you had in mind?

Reply:
I'll find some examples but there should be some in `stanford_basic`

Reply:
Ok. I looked some more and tried again. I think it is ok. I also realized that the soe_basic.theme file has references to stanford_basic. Should I change them all to soe_basic? Or is it better to share?

Reply:
soe_basic is defined as a subtheme for stanford_basic, so I think that reference may be necessary elsewhere, since it's storing the location of the parent theme.